mypyc/test-data/run-classes.test

```diff
@@ -2887,3 +2887,66 @@ def test_function():
     explicit_ext_inst = AnnotatedExtensionClass()
     with assertRaises(AttributeError):
         setattr(explicit_ext_inst, 'attr_instance', 6)
+
+[case testMypycAttrNativeClassDunder]
+from mypy_extensions import mypyc_attr
+from typing import Generic, TypeVar
+
+_T = TypeVar("_T")
+
+@mypyc_attr(native_class=False)
+class Bar(Generic[_T]):
+    # Note the lack of __deletable__
+    def __init__(self) -> None:
+        self.value: str = 'start'
+    def __get__(self, instance: _T, owner: type[_T] | None = None) -> str:
+        return self.value
+    def __set__(self, instance: _T, value: str) -> None:
+        self.value = value
+    def __delete__(self, instance: _T) -> None:
+        del self.value
+
+@mypyc_attr(native_class=False)
+class Foo(object):
+    bar: Bar = Bar()
+
+[file driver.py]
+import native
+
+f = native.Foo()
+assert(hasattr(f, 'bar'))
+assert(f.bar == 'start')
+f.bar = 'test'
+assert(f.bar == 'test')
```

Comment:
Similarly here it would be good to ensure that `__get__` (and `__delete__` below) actually get called.